WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostPagerActivity.java

```diff
@@ -218,6 +224,20 @@ public void onBackPressed() {
     }
 
     /*
+     * "bumps" the pageview for the post at the passed position if it hasn't already been done
+     */
+    private void bumpPageViewIfNeeded(int position) {
```

Comment:
This method seems to "bump" pageview for the post at the passed position if it hasn't already done.

What happens if I'm using the reader and new articles come in the list? I suppose the old article that I've already seen changes its position in the list, so if I visit it again the stats are bumped once again. Am I wrong?